src/lib/worker.js

```diff
@@ -26,6 +26,7 @@ const init = async() => {
     hostIp = await lookup(hostName);
     hostIp = hostIp.address;
     hostIpAndPort = hostIp + ':' + config.mongoPort;
+    console.info('Running on host '+hostIpAndPort+' hostname '+hostName);
```

Comment:
Could you insert spaces here?
```suggestion
    console.info('Running on host ' + hostIpAndPort + ' hostname ' + hostName);
```

There are a few more lines